components/Comments.jsx

```diff
@@ -19,15 +21,34 @@ const Comments = ({ articleid }) => {
     fetchComments();
   }, [articleid]);
 
+  const isUserAuthorized = (commentAuthor) => {
+    return user && user.username === 'tickle122' && user.username === commentAuthor;
+  };
+
+  const handleDeleteComment = async (commentId) => {
+    try {
+      await deleteCommentById(commentId); 
+      setArticleComments(articleComments.filter(comment => comment.comment_id !== commentId));
+    } catch (error) {
+      console.error("Error deleting comment:", error);
```

Comment:
It's great that you have some code which reacts to an error happening with the API request to fetch a comment. You could use this to provide some feedback to the user, to indicate there's been a problem, rather than just console logging the error.